PowerApps/Invoice Automation/Azure-FunctionAppDeployment.ps1

```diff
@@ -0,0 +1,33 @@
+#http://www.redbaronofazure.com/?cat=428
+#https://demiliani.com/2020/03/26/a-quick-way-to-deploy-your-azure-functions-in-the-cloud/
+
+
+$location = "East US"
+$resourceGroupName = "InvoiceAutomation"
+$storageAccountName = "invoicenautomationrg"
+$functionName = "InvoiceLineItems"
+$sourceZipPath = "C:\Users\stpuceli\source\repos\HelloWorld\bin\Release\netcoreapp3.0\publish\InvoiceLineItems.zip"
```

Comment:
This is a path on your machine, can you add instructions to unzip the relevant file to a path and then set here please